vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
+### Step 4: Login to Keycloak
 
-Now let's give the admin user the correct role. 
+1. Go to <http://localhost:9080> 
+2. Login using the the default credentials of `username: admin`, `password: admin`
 
-4. Login to keycloak on <http://localhost:9080> using the `admin/admin` credentials. 
+### Step 5: Create the `conference-user` and `conference-admin` roles 
 
-First we need to create the two roles per our requirements. We're going to add the roles to the `internal` client because it's the one configured by default in the Spring Boot application.yml.
+Add the `conference-user` and `conference-admin` roles to the `internal` client. 
           
-5. Go to `Clients → internal → Roles` and click `Add Role`
-6. Fill in the `Role Name` with `conference-admin` and click `Save`
-7. Repeat steps 5-6 to create the `conference-user` role. 
+1. Go to `Clients` → `internal` → `Roles`
+2. Click `Add Role`
+3. Fill in the `Role Name` with `conference-admin`
+4. Click `Save`
+5. Repeat these steps to create the `conference-user` role
 
-Now we need to map this role to our user.
+> Note: The `internal` client is configured by default in the Spring Boot `application.yml`.
 
-8. Go to `Users → View all users → admin → Role Mappings`
-9. Select `internal` for the `Client Roles` and then move `conference-user` from `Available Roles` to `Assigned Roles`
-10. Go back to the MFE and you should now see the full list of Conferences.
+### Step 6: Map the `conference-user` role to the admin user
 
-We've now successfully secured the `getAllConferences` API but we have more to do. The admin user was granted just the `conference-user` role but still has access to delete Conferences. We need to lock that down.
+To secure the `getAllConferences` API:
 
-11. Go back into the `ConferenceResource.java` file and add this annotation to the `deleteConference` method:
+1. Go to `Users` → `View all users` → `admin` → `Role Mappings`
+2. Select `internal` for the `Client Roles` 
+3. Move `conference-user` from `Available Roles` to `Assigned Roles`
+4. Return to the MFE to confirm you see the full list of Conferences
 
+### Step 7: Restrict the ability to delete Conferences
+
+The `conference-user` role grants the admin user permission to delete Conferences. To restrict the delete method to the `conference-admin` role:
+
+1. Go to the `src/main/java/com/<ORG>/<NAME>.web.rest` directory
+2. Open `ConferenceResource.java` 
+3. Modify the `deleteConference` method by adding the following annotation:
 ```
     @PreAuthorize("hasAuthority('conference-admin')")
-    public ResponseEntity<Void> deleteConference(@PathVariable Long id) {
+    public ResponseEntity<Void> deleteConference(@PathVariable Long ID) {
```

Comment:
Was it caps in the generated code? It was not in the previous version and caps doesn't match the standard java style.

Reply:
i got carried away with correcting lowercase "id"...